manifests/2.3.0/opensearch-2.3.0.yml

```diff
@@ -10,7 +10,7 @@ ci:
 components:
   - name: OpenSearch
     repository: https://github.com/opensearch-project/OpenSearch.git
-    ref: 2.x
+    ref: 2.3
```

Comment:
Encapsulate it in quotes to make it a string.`'2.3'`

Reply:
Updated.